config/config_sms.nurse_sms.qa.json.example

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
These config file names don't match with the app names. But I personally don't think we should have these example configs at this time (or ever - they tend to just lag behind and therefore be unhelpful).